### DISCUSSION TITLE
Best way to create and return the created object?

### DISCUSSION THREAD

**Discussion by saikarthikp9** [prisma/prisma#17604] [Q&A] [answered]
Best way to create and return the created object?

**Comment by nurul3101** [comment, marked as answer]
Hello @saikarthikp9 👋 

When a record is created using the `create` method, the created record is returned in response.
Here's a sample example, considering this schema:

```prisma
generator client {
  provider        = "prisma-client-js"
}

datasource db {
  provider = "postgresql"
  url      = env("DATABASE_URL")
}

model User {
  id        Int      @id @default(autoincrement())
  email     String   @unique
  name      String
  createdAt DateTime @default(now())
  updatedAt DateTime @updatedAt
}

```

This script would create a user and return the created user in response:

```typescript
import { PrismaClient } from '@prisma/client';

const prisma = new PrismaClient();

async function main() {
  const createdUser = await prisma.user.create({
    data: {
      email: 'test@prisma.io',
      name: 'Test',
    },
  });

  console.log({ createdUser });
}

main()
  .catch((e) => {
    throw e;
  })
  .finally(async () => {
    await prisma.$disconnect();
  });
```

Here's the response:
<img width="549" alt="Screenshot 2023-01-30 at 2 48 20 PM" src="https://user-images.githubusercontent.com/13049676/215436417-6fb38774-213a-4f24-83ae-408ccc2e2f74.png">

Please let me know in case you have any further queries.

**Reply by saikarthikp9** [reply]
@nurul3101 , Thanks, this is working as expected. I was just not awaiting the response.

**Reply by nurul3101** [reply]
I am glad to hear that! 🎉

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
